### PR TITLE
{chem}[iccifort/2019.5.281] NCIPLOT v4.0-20190718

### PR DESCRIPTION
**Pull request by smoors**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node376.hydra.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/fe6b9b626ae89c45b1d92d09572ac1c6 for a full test report.

File: easybuild/easyconfigs/n/NCIPLOT/NCIPLOT-4.0-20190718-iccifort-2019.5.281.eb [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+modextrapaths = {'NCIPLOT_HOME': ''}
+
+sanity_check_paths = {
+    'files': ['bin/nciplot'],
+    'dirs': ['dat'],
+}
+
+# site-specific: set OMP_NUM_THREADS equal to number of available cores
+modluafooter = 'if os.getenv("PBS_NUM_PPN") ~= nil then pushenv("OMP_NUM_THREADS", os.getenv("PBS_NUM_PPN")) end'
```
Comment (akesandgren) [inline easybuild/easyconfigs/n/NCIPLOT/NCIPLOT-4.0-20190718-iccifort-2019.5.281.eb]: This part doesn't really belong here. It's the responsibility of the user to set OMP_NUM_THREADS.
(And the responsibility of the site documentation to tell them how and when :-)

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/f041c95b7016eb3a971f894084776a0e for a full test report.

**Comment by migueldiascosta** [comment]
Going in, thanks @smoors!

**Pull request merged**